index.js

```diff
@@ -216,6 +216,9 @@ function loadConfig(program) {
         return
       }
 
+      // Do NOT write this to the config file. It's meant as a one-time override.
+      parsedConfig.sendUserAgent = program.sendUserAgent || process.env.LOGGER_SEND_USER_AGENT
```

Comment:
@darinspivey, this is fine but why setting it to `parsedConfig.sendUserAgent` but not directly to `config.sendUserAgent`? Is there any specific reason?

Another question is how `program.sendUserAgent` is gonna get used? What is the key? I mean, for tags, we have `-t` or `--tag` but how can that information come from `program` arguments? Because agent isn't like the code library in which you can add parameters while initiating

If there is a way to pass this information thru cli arguments, can you please update the `README.md` too?

Reply:
Yep, I can move the setting to the `config` object creation below.  Good call.

`program.sendUserAgent` is the "config-driven" approach that would be used in testing.  I'm using it in the test that was just made for this where it passed it directly.  The goal is to not have to use the env var because testing with those is a one-shot deal.

I need to update the README, thanks.

Reply:
The README has no information about the env vars.  This is one of the things that would completely need re-worked.  If I add this, then we'd have to add all of the env vars, and I think that's out of scope.  I think we should decide as a group whether or not we should spend time there.

Reply:
Thanks, @darinspivey! It all makes sense